server/protocols/webadmin-cli/src/main/java/org/apache/james/httpclient/JwtToken.java

```diff
@@ -19,18 +19,40 @@
 
 package org.apache.james.httpclient;
 
+import java.io.File;
+import java.io.PrintStream;
 import java.util.Optional;
+import java.util.Scanner;
 
 public class JwtToken {
 
     private final Optional<String> jwtTokenString;
 
-    public JwtToken(Optional<String> jwtTokenString) {
-        this.jwtTokenString = jwtTokenString;
+    public JwtToken(Optional<String> jwtTokenString, Optional<String> jwtFilePath, PrintStream err) {
+        this.jwtTokenString = handleJwtOptions(jwtTokenString, jwtFilePath, err);
     }
 
     public Optional<String> getJwtTokenString() {
         return this.jwtTokenString;
     }
 
+    private Optional<String> handleJwtOptions(Optional<String> jwtTokenString, Optional<String> jwtFilePath, PrintStream err) {
+        return jwtFilePath.map(path -> {
+                try {
+                    File myObj = new File(jwtFilePath.get());
+                    Scanner myReader = new Scanner(myObj);
+                    StringBuilder data = new StringBuilder();
+                    while (myReader.hasNextLine()) {
+                        data.append(myReader.nextLine());
+                    }
+                    myReader.close();
+                    return Optional.of(data.toString());
+                } catch (Exception e) {
+                    e.printStackTrace(err);
+                    return jwtTokenString;
```

Comment:
Here instead of failing we are falling back to no authentication. Is this intentional? Is there a test about this?

Reply:
Yes.

My intention is making JwtToken pre-process two options input( maybe people want to test type --jwt-token and --jwt-from-file at the same time?) and then the output is only one final Optional<String> jwtTokenString

Then FeignClientFactory will only have to process 1 Optional<String>

Reply:
Do you have a test about it?